cmd/dashboard.go

```diff
@@ -43,11 +46,39 @@ var DashboardCmd = &cobra.Command{
 			localPort = port
 		}
 
-		config, err := kubernetes.GetKubeConfig()
-
+		config, client, err := kubernetes.GetKubeConfigClient()
 		if err != nil {
 			print.FailureStatusEvent(os.Stdout, "Failed to initialize kubernetes client")
-			return
+			os.Exit(1)
+		}
+
+		// search for dashboard service namespace in order:
+		// user-supplied namespace, dapr-system, default
+		namespaces := []string{dashboardNamespace}
+		if dashboardNamespace != defaultNamespace {
+			namespaces = append(namespaces, defaultNamespace)
+		}
+		namespaces = append(namespaces, "default")
```

Comment:
if users provides -n default.
This will add "default" again. It should be added inside if similar to the previous one.